packages/mongo-nav/src/data.ts

```diff
@@ -114,11 +114,21 @@ export const urlFixtures: Required<URLSInterface> = {
   },
 };
 
+export function getProductHost(today = new Date(Date.now())) {
```

Comment:
We're only using this in one place for the next week or two. I don't think we need to make `today` configurable.

Reply:
The reason to do this is to make it testable. Followed the same convention used in ProjectNav

Reply:
I don't think this is an effective test for how we're using the function.

You might consider overwriting the global Date constructor in the test to set it to the desired dates, and using the function with no parameter like it is in the code.

Reply:
https://stackoverflow.com/questions/8922145/overriding-the-javascript-date-constructor